lib/models/user_model.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:json_annotation/json_annotation.dart';
+
+/// Class with represents the user model.
+///
+/// Holds all data pertaining to a single user.
+@JsonSerializable(explicitToJson: true)
+class UserModel {
+  String? netID;
+  String? displayName;
+  String? userName;
+  // String? email;
+  // String? college;
```

Comment:
Could you remove the commented-out lines of code here and in the rest of the file?